extensions/rendering-hints/README.md

```diff
@@ -0,0 +1,63 @@
+
+This document explains the fields of the STAC Rendering Hints (`render`) Extension to a STAC Item. They are meant
+to be 'hints' - things that could be discovered by inspecting the data, but provided in STAC to help out tools
+that can take advantage of them. It is written with geospatial raster data in mind, but could apply to other data types.
+
+The field names defined herein should be added as fields in the Item Properties object. When specified on an Item, 
+the values are assumed to apply to all non-thumbnail Assets in that Item. 
+
+## Examples
+
+- TODO
+
+## Schema
+
+- TODO
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| render:max_zoom  | integer | **REQUIRED.** The maximum [zoom level](https://wiki.openstreetmap.org/wiki/Zoom_levels), expressed as an integer, that corresponds with the highest resolution of the data. Specified as [Web Mercator](https://en.wikipedia.org/wiki/Web_Mercator_projection) zoom levels. |
+| render:min_zoom  | integer  | The minimum [zoom level](https://wiki.openstreetmap.org/wiki/Zoom_levels), expressed as an integer, that tells renderers where the overviews of the data stop. |
+| render:type    | string | The data `type` (float, int, complex, etc) to let the renderer apply any needed rescaling up front. The full set of options is listed below. |
+
+**render:max_zoom**: This field informs renderers of the resolution of the data, using 
+[Web Mercator](https://en.wikipedia.org/wiki/Web_Mercator_projection) zoom levels (since it is used in the majority
+of web tile maps). Some renderers will not use web mercator, but most should be able to use the hint in web mercator 
+and translate to the appropriate zoom level. The easiest way to get the zoom levels for data is to use [rio-cogeo](https://pypi.org/project/rio-cogeo/): `[get_zooms()](https://github.com/cogeotiff/rio-cogeo/blob/master/rio_cogeo/utils.py#L69)` in python or using `[rio info](https://github.com/cogeotiff/rio-cogeo/#examples)` on the command line. Generally will
+be between 
+
+**render:min_zoom** - This field informs renderers of the minimum level supported by the data, generally corresponding
+to the resolution for which overviews were generated. It works just like max zoom: specified in web mercator and 
+rio-cogeo can be used to easily get the value. If not specified then renderers will assume the minimum zoom is 0.
+
+**render:type** - Specifies the data storage `type` of the assets. Can be used at the asset level if the assets are of
+different types. This is used to let the renderer know if any type of rescaling is needed up front. The possible values
+for the type in STAC are specified in the table below.
+
+| Type Name | Description |
+|-----------|-------------|
+| `unknown` | Not known   |
+| `byte`    | An unsigned 8-bit integer (common for 8-bit rgb png's) |
```

Comment:
as far as type names go, `int8` is numpy's reference to the type:

```
In [1]: import numpy as np

In [2]: np.int8
Out[2]: numpy.int8

In [3]: np.byte
Out[3]: numpy.int8
```

And `rasterio` has a similar idea:

```
In [1]: import rasterio

In [2]: src = rasterio.open("<a tiff i have>")

In [3]: src.dtypes
Out[3]: ('uint8', 'uint8', 'uint8')
```

I don't think it's super important but it's not an uncommon name for the type

Reply:
Oh no I scrolled up in gitter and now see the numpy / rasterio discussion 🙃

Reply:
I really know very little about these things, so I'm happy for whatever you all think is best. I just want to have a clear list for people to pick from.

Reply:
Oh sorry; I thought `byte` was an alias for `uint8`, not `int8`. I see Numpy has `ubyte` as the alias for `uint8`. `uint8` is the more common one for imagery as you saw

Reply:
So I should change this to just be unit8 ?

Reply:
It doesn't hurt to include both I guess

| Type Name | Description |
|-----------|-------------|
| `uint8` | 8-bit unsigned integer |
| `int8` | 8-bit signed integer |